horovod/spark/common/store.py

```diff
@@ -248,7 +249,7 @@ def get_run_path(self, run_id):
         return os.path.join(self.get_runs_path(), run_id)
 
     def get_checkpoint_path(self, run_id):
-        return os.path.join(self.get_run_path(run_id), self.get_checkpoint_filename()) \
+        return self.get_run_path(run_id) \
```

Comment:
This is actually conflicting with my on-going work for some Keras/TF1 checkpoint stuff at Uber, since I'm calling this interface directly to get file name.
But I'm fine to add additional path join function to compose file name.

Reply:
```get_checkpoint_path()``` is used for all the estimators to check if the checkpoints exists, plus to fetch the checkpoint( except for lightning), if we yield the whole path here then all these functionalities can't be fulfilled hence returning the run_path